### ISSUE TITLE
Use icons from HA?

### ISSUE THREAD

**Issue by dummptyhummpty** [gadgetchnnel/lovelace-home-feed-card#12]
Still loving this card, so thanks for that! Is it possible to use the icons from HA instead of having to override them? For many of the entities I want to track I have an icon based on state, i.e. a door open or closed, window open or closed, etc. Previously, I was able to accomplish this with the auto-entities card, but this gets complicated with the include_history feature.

**Comment by gadgetchnnel** [comment]
@dummptyhummpty I've spotted an issue where the default icon for a particular sensor isn't picked up when using the include_history option. I believe it is trying to pick up the icon from the history data, but the history doesn't actually have the icon.
It should be a fairly easy fix, so I'll sort it when I have time.

**Comment by dummptyhummpty** [comment]
@gadgetchnnel Ok, cool. I'll keep an eye out for that change and report back.

**Comment by gadgetchnnel** [comment]
@dummptyhummpty I found that this was actually affecting more than the history, so the change was more complex than I thought, but I have implemented this in the 0.2.10b1 beta release.

It now uses the same method of displaying the icon (state-badge) as the **entities** card. This does mean that some of the icons may display differently:

1. Unless the icons are overridden the icon may change based on the state (e.g. an door sensor which is open will display a different icon from one which is closed)
2. The colour of the icon will also change based on the state, whether or not the icon is overridden. The colours will depend on your theme (I have the default theme and the icon is blue if the sensor is off and yellow if it is on).
